client/my-sites/customer-home/cards/features/stats-v2/index.jsx

```diff
@@ -0,0 +1,250 @@
+import {
+	getMostPopularDatetime,
+	getTopPostAndPage,
+	getViewAndVisitors,
+	isLoadingStats,
+} from './utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const placeholderChartData = times( 7, () => ( {
+	value: Math.random(),
+} ) );
+
+export const StatsV2 = ( {
+	chartData,
+	chartQuery,
+	insightsQuery,
+	isLoading,
+	isSiteUnlaunched,
+	mostPopularDay,
+	mostPopularTime,
+	siteId,
+	siteSlug,
+	topPage,
+	topPost,
+	topPostsQuery,
+	views,
+	visitors,
+	visitsQuery,
+} ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	useEffect( () => {
+		if ( ! isSiteUnlaunched ) {
+			dispatch( requestChartCounts( chartQuery ) );
+		}
+	}, [ isSiteUnlaunched ] );
+
+	const heading = (
+		<h2 className="stats-v2__heading customer-home__section-heading">
+			{ translate( 'Stats at a glance' ) }
+		</h2>
+	);
+
+	if ( isSiteUnlaunched ) {
+		return (
+			<>
+				{ heading }
+				<Card className="stats-v2">
+					<Chart data={ placeholderChartData } isPlaceholder>
+						<div>
+							{ translate( 'Launch your site to see a snapshot of traffic and insights.' ) }&nbsp;
+							<InlineSupportLink
+								supportPostId={ 4454 }
+								supportLink={ localizeUrl( 'https://wordpress.com/support/stats/' ) }
+								showIcon={ false }
+								text={ translate( 'Learn about stats.' ) }
+								tracksEvent="calypso_customer_home_stats_support_page_view"
+								statsGroup="calypso_customer_home"
+								statsName="stats_learn_more"
+							/>
+						</div>
+					</Chart>
+				</Card>
+			</>
+		);
+	}
+
+	const showTopPost = !! topPost;
+	const showTopPage = !! topPage;
+	const showViews = ! showTopPost || ! showTopPage;
+	const showVisitors = ! showTopPost && ! showTopPage;
+
+	return (
+		<>
+			{ heading }
+			<Card className="stats-v2">
+				<QuerySiteStats siteId={ siteId } statType="statsVisits" query={ visitsQuery } />
+				<QuerySiteStats siteId={ siteId } statType="statsInsights" query={ insightsQuery } />
+				<QuerySiteStats siteId={ siteId } statType="statsTopPosts" query={ topPostsQuery } />
+				{ isLoading ? (
+					<Chart data={ placeholderChartData } isPlaceholder>
+						<Spinner />
+					</Chart>
+				) : (
+					<>
+						<CardHeading>{ translate( 'Views' ) }</CardHeading>
+						<Chart data={ chartData } />
+						<div className="stats-v2__data">
+							<div className="stats-v2__data-item">
+								<div className="stats-v2__data-label">{ translate( 'Most popular day' ) }</div>
+								<div className="stats-v2__data-value">{ mostPopularDay }</div>
+							</div>
+							<div className="stats-v2__data-item">
+								<div className="stats-v2__data-label">{ translate( 'Most popular hour' ) }</div>
+								<div className="stats-v2__data-value">{ mostPopularTime ?? '-' }</div>
+							</div>
+							{ showTopPost && (
+								<div className="stats-v2__data-item">
+									<div className="stats-v2__data-label">{ translate( 'Top post' ) }</div>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **83** times:
`translate( 'Top Posts' )` *ES Score: 7*